lib/server.ts

```diff
@@ -82,6 +118,22 @@ export function createApp(devJsonPath: string, opts: DevOptions): Express.Applic
       })
   });
 
+  /**
+   * Adds the loader json
+   */
+  app.post("/api/apps", (req, res) => {
+    console.log(`Request: ${JSON.stringify(req.body)}`);
```

Comment:
Replace log, 

log.info("adding ${srcs} to watch list...")

or similiar.

Then let's do a confirmation on line 130

log.info("New apps added successfully.")

Reply:
Maybe if we can do something better than srcs and just did the core app account of what is being added